wasm-wrappers/src/lib.rs

```diff
@@ -63,6 +63,7 @@ impl From<Network> for ChainType {
     }
 }
 
+/// Indicates whether a token an be frozen
```

Comment:
token can

Reply:
Done. Thanks for proof reading everything!